--- php-pecl-imagick-3.0.1/config.m4~	2010-11-18 22:16:01.000000000 +0100
+++ php-pecl-imagick-3.0.1/config.m4	2013-06-13 10:04:07.872529471 +0200
@@ -52,8 +52,8 @@
   else	
     AC_MSG_CHECKING(for MagickWand.h header file)
 
-    if test -r $WAND_DIR/include/ImageMagick/wand/MagickWand.h; then
-      AC_MSG_RESULT(found in $WAND_DIR/include/ImageMagick/wand/MagickWand.h)
+    if test -r $WAND_DIR/include/ImageMagick-6/wand/MagickWand.h; then
+      AC_MSG_RESULT(found in $WAND_DIR/include/ImageMagick-6/wand/MagickWand.h)
     else
       AC_MSG_ERROR(Cannot locate header file MagickWand.h)
     fi
